examples/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md

```diff
@@ -22,39 +22,42 @@ This Terraform example creates a ROSA STS cluster that uses a managed OIDC confi
         ```
         export TF_VAR_account_role_prefix=<account_role_prefix>
         ```
-    2. The availability zones should be a zone within your AWS region, which you chose with the `TF_VAR_cloud_region` variable. This value must be enclosed by double quotes, a bracket pair, and single quotes.     
+    1. The availability zones should be a zone within your AWS region, which you chose with the `TF_VAR_cloud_region` variable. This value must be enclosed by double quotes, a bracket pair, and single quotes.
         ```    
         export TF_VAR_availability_zones='["<aws_az_region"]' 
         ```
-    3. This value sets the AWS region where your cluster launches.
+    1. This value sets the AWS region where your cluster launches.
         ```
         export TF_VAR_cloud_region=<aws_region_name>
         ```
-    4.  This value is your cluster name. The cluster name cannot exceed 15 characters or the Terraform apply fails.  
+    1.  This value is your cluster name. The cluster name cannot exceed 15 characters or the Terraform apply fails.  
```

Comment:
I think perhaps the "Terraform apply fails" should either be "the Terraform apply command fails" or "the `terraform apply` fails.